src/annotation.js

```diff
@@ -66,8 +67,15 @@ export class Annotation extends Modifier {
 
     let width = 0;
     for (let i = 0; i < annotations.length; ++i) {
+      let testWidth = 0;
       const annotation = annotations[i];
-      width = Math.max(annotation.getWidth(), width);
+      const textFont = TextFont.getTextFontFromVexFontData({ family: annotation.font.family,
+        size: annotation.font.size, weight: 'normal' });
+      textFont.setFontSize(annotation.font.size);
+      for (let j = 0; j < annotation.text.length; ++j) {
+        testWidth += textFont.getWidthForCharacter(annotation.text[j]) * (72 / 96);
```

Comment:
why 72/96? this is 3/4. if you don't want to explain the magic number, you can just write * 0.75 ;)

Reply:
i'd at least explain where the number comes from in a comment, if not create named variables for the constituents.

Reply:
Is it really just 3/4?  I feel silly :).  It's points to pixels, but really the font code should return pixels. And I don't like the way you have to set the font size after sending it in, let me clean this up a bit.

Reply:
Yes, a comment here would be helpful.

Reply:
It was a mistake, copied from a smoosic calculation (which was also wrong).  I removed it.